geowebcache/swiftblob/pom.xml

```diff
@@ -0,0 +1,57 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <!-- mvn -Dtest=StorageBrokerTest -Dmaven.test.jvmargs='-XX:+HeapDumpOnOutOfMemoryError -Xms32m -Xmx32m' test -->
+    <modelVersion>4.0.0</modelVersion>
+    <properties>
+        <jclouds.version>2.2.0</jclouds.version>
+    </properties>
+
+    <parent>
+        <groupId>org.geowebcache</groupId>
+        <artifactId>geowebcache</artifactId>
+        <version>1.18-SNAPSHOT</version><!-- GWC VERSION -->
+    </parent>
+
+    <groupId>org.geowebcache</groupId>
+    <artifactId>gwc-swift</artifactId>
+    <packaging>jar</packaging>
+    <name>Swift Storage</name>
+    <url>http://geowebcache.org</url>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>org.geowebcache</groupId>
+            <artifactId>gwc-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <!-- jclouds OpenStack dependencies -->
+        <dependency>
+            <groupId>org.apache.jclouds.api</groupId>
+            <artifactId>openstack-swift</artifactId>
+            <version>${jclouds.version}</version>
```

Comment:
We try and add version numbers to the root pom.xml dependency management section; something to consider for later.